packages/babel-plugin/src/utils/ast-builders.tsx

```diff
@@ -193,15 +193,15 @@ const styledTemplate = (opts: StyledTemplateOpts, meta: Metadata): t.Node => {
     ${propsToDestructure.map((prop) => prop + ',').join('')}
     ...props
   }, ref) => (
-    <CC>
-      <CS ${nonceAttribute}>{%%cssNode%%}</CS>
-      <C
-        {...props}
-        style={%%styleProp%%}
-        ref={ref}
-        className={ax(["${opts.classNames.join(' ')}", props.className])}
-      />
-    </CC>
+      <>
+        <style ${nonceAttribute}>{%%cssNode%%}</style>
```

Comment:
remove fragment + style as well

Reply:
ahh. hehe. I just started. Lol. Sure. I am exploring more in css file generation.

Reply:
I just wanted to see how storybook deviates for now.

Reply:
ah sorry jumped the gun 😆 

saw a email and couldnt help myself - will :homesimpsonintothebushes: until tomorrow